src/organisation/components/pba-numbers-form/pba-numbers-form.component.ts

```diff
@@ -0,0 +1,141 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { AbstractControl, FormArray, FormBuilder, FormControl, FormGroup, ValidatorFn, Validators } from '@angular/forms';
+import { RxwebValidators } from '@rxweb/reactive-form-validators';
+import { UpdatePbaNumbers } from '../../../organisation/models/update-pba-numbers.model';
+
+@Component({
+  selector: 'app-prd-pba-numbers-form-component',
+  templateUrl: './pba-numbers-form.component.html',
+})
+export class PbaNumbersFormComponent implements OnInit {
+
+  public static title = 'Add or remove PBA accounts';
+
+  public pbaFormGroup: FormGroup;
+  public summaryErrors: {
+    header: string;
+    isFromValid: boolean;
+    items: {
+      id: string;
+      message: any;
+    }[]
+  };
+
+  @Input()
+  public updatePbaNumbers: UpdatePbaNumbers;
+
+  constructor(private readonly fb: FormBuilder) { }
+
+  ngOnInit() {
+    this.initialiseForm();
+  }
+
+  get pbaNumbers(): FormArray {
+    return this.pbaFormGroup.get('pbaNumbers') as FormArray;
+  }
+
+  public onAddNewBtnClicked(): void {
+    this.pbaNumbers.push(this.newPbaNumber());
+  }
+
+  public onRemoveNewPbaNumberClicked(i: number): void {
+    this.pbaNumbers.removeAt(i);
```

Comment:
If it's removing exising PBAs, could you also put this in the ngrx/store.
If it's new added PBAs it can be removed from the formGroup directly.